src/styles/global.css

```
@@ -23,6 +23,9 @@ h1 {
 /* nav styles */
 
 .hamburger {
+  background: transparent;
+  display: block;
+  border: none;
   padding-right: 20px;
   cursor: pointer;
 }

```

```suggestion
}

.dark .hamburger {
  background-color: #fff;
  color: #000;
}
```

We tweak the appearance of this button in dark mode.

We would also need to remove the CSS for the previous lines (`.hamburger .line`) which are right below but I cannot suggest it as it's outside of the review range.

---

And the class name is still `hamburger`... :sweat_smile: